docs/en/sql-reference/sql-statements/Administration/ADMIN COMPACT.md

```diff
@@ -0,0 +1,52 @@
+---
```

Comment:
New doc need to be added to the `docs/.vuepress/sidebar/en.js` and `zh-CN.js`

Reply:
> New doc need to be added to the `docs/.vuepress/sidebar/en.js` and `zh-CN.js`

Done.